espnet/nets/pytorch_backend/e2e_asr.py

```diff
@@ -101,19 +96,6 @@ def __init__(self, idim, odim, args, asr_model=None, mt_model=None):
         # speech translation related
         self.replace_sos = args.replace_sos
 
-        if args.use_frontend:
```

Comment:
I actually could not fully track where the front end processing is performed. Could you point out it to me?
I prefer to have simple sequential processing in the main training loop.

Reply:
> I actually could not fully track where the front end processing is performed. Could you point out it to me?

I changed my mind. 
I put back to frontend class in E2E and changed the interface slightly. Maybe you can get the procedure now.

```python
frontend = FrontendASR(idim, args)
asr = E2E(frontend.featdim, odim, args)
asr.register_frontend(frontend)
```